cocos/platform/android/jni/JniHelper.h

```diff
@@ -69,6 +69,11 @@ class CC_DLL JniHelper
     static jobject classloader;
     static std::function<void()> classloaderCallback;
 
+
+    /**
+    @brief Check whether there are proper JniMethodInfo
```

Comment:
Why check? It's call of Java static void method.

Reply:
thanks.